### ISSUE TITLE
How to reproduce VOC07 linear classification result?

### ISSUE THREAD

**Issue by yxchng** [facebookresearch/barlowtwins#32]
I used vissl to reproduce VOC07 linear classification result using the pre-trained model linked in this repo and the following config (default config in vissl with LINEAR_EVAL_FEAT_POOL_OPS_MAP removed since we only want to test the last layer features). The accuracy I got is 86.830912% which is quite a bit higher than the 86.2% reported in your paper. However, for multi-crop SwAV, I am able to reproduce the 88.9% figure using this config. Can you kindly comment on the settings you use? What are differences from the config above? Thanks.

```
# @package _global_
config:
  DATA:
    NUM_DATALOADER_WORKERS: 5
    TRAIN:
      DATA_SOURCES: [disk_filelist]
      LABEL_SOURCES: [disk_filelist]
      DATASET_NAMES: [voc2007]
      BATCHSIZE_PER_REPLICA: 32
      TRANSFORMS:
        - name: Resize
          size: 256
        - name: CenterCrop
          size: [224, 224]
        - name: ToTensor
        - name: Normalize
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
      MMAP_MODE: False
      COPY_TO_LOCAL_DISK: False
      COPY_DESTINATION_DIR: /tmp/voc2007/
    TEST:
      DATA_SOURCES: [disk_filelist]
      LABEL_SOURCES: [disk_filelist]
      DATASET_NAMES: [voc2007]
      BATCHSIZE_PER_REPLICA: 32
      TRANSFORMS:
        - name: Resize
          size: 256
        - name: CenterCrop
          size: [224, 224]
        - name: ToTensor
        - name: Normalize
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
      MMAP_MODE: False
      COPY_TO_LOCAL_DISK: False
      COPY_DESTINATION_DIR: /tmp/voc2007/
  MODEL:
    WEIGHTS_INIT:
      PARAMS_FILE: "specify path"
      STATE_DICT_KEY_NAME: classy_state_dict
      # STATE_DICT_KEY_NAME: model_state_dict
    FEATURE_EVAL_SETTINGS:
      EVAL_MODE_ON: True
      FREEZE_TRUNK_ONLY: True
      EXTRACT_TRUNK_FEATURES_ONLY: True
    TRUNK:
      NAME: resnet
      RESNETS:
        DEPTH: 50
  DISTRIBUTED:
    NUM_NODES: 1
    NUM_PROC_PER_NODE: 8
  MACHINE:
    DEVICE: gpu
  CHECKPOINT:
    DIR: .
    OVERWRITE_EXISTING: False
  SVM:
    costs:
      costs_list: [0.01, 0.1, 1.0, 2, 5, 10, 15, 20, 50, 100, 1000]
    normalize: True
    loss: squared_hinge
    penalty: l2
    dual: True
    max_iter: 2000
    cross_val_folds: 3
    force_retrain: False
```

I am attaching the following screenshot from your paper as a reference.

<img width="665" alt="Screenshot 2021-10-03 at 9 07 16 PM" src="https://user-images.githubusercontent.com/10518587/135755656-f472df31-b40c-4d27-88e1-597698f99be8.png">
